moniker_src/data_moniker.proto

```diff
@@ -1,30 +1,17 @@
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
 syntax = "proto3";
 
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
-import "google/protobuf/any.proto";
+option cc_enable_arenas = true;
+option csharp_namespace = "NationalInstruments.DataMonikers";
 
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
-package ni.data_monikers.v1;
+package ni.data_monikers;
 
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
-option csharp_namespace = "NationalInstruments.DataMonikers";
+import "google/protobuf/any.proto";
 
-//---------------------------------------------------------------------
-//---------------------------------------------------------------------
-service MonikerService {
+service DataMoniker {
```

Comment:
I think this name will conflict as we have the same name for the class **DataMonikerService**

Reply:
@ccifra I noticed that service definitions in the proto files do not have "Service" appended at the end in the grpc-device repo. e.g daq has NiDAQmx as the service name and similarly for session_utilities and debugsessionproperties_restricted and so on.
https://github.com/ni/grpc-device/blob/main/generated/nidaqmx/nidaqmx.proto

From that perspective, it should be fine to keep the "service" keyword in the class name instead of the service name in the proto

Reply:
I think for new things we should follow conventions.  grpc-sideband will be used for more than just grpc-device.
for example
https://github.com/ni/ni-apis/blob/main/ni/measurementlink/measurement/v2/measurement_service.proto